doc/images.md

```diff
@@ -192,25 +196,24 @@ class MyGame extends Game {
 }
 ```
 
-A better alternative to generate a list of sprites is to use the `sequenced` constructor:
+A better alternative to generate a list of sprites is to use the `fromFrameData` constructor:
 
 ```dart
   const amountOfFrames = 8;
-  SpriteAnimation a = SpriteAnimation.sequenced(imageInstance, amountOfFrames, textureWidth: 16.0);
+  SpriteAnimation a = SpriteAnimation.fromFrameData(
+    imageInstance,
+    SpriteAnimationFrame.sequenced(
+      amount: amountOfFrames,
+      textureSize: Vector2(16.0, 16.0),
+      stepTime: 0.1,
+    ),
+  );
 ```
 
-In which you pass the file name, the number of frames and the sprite sheet is automatically split for you according to the 4 optional parameters:
-
-* textureX : x position on the original image to start (defaults to 0)
-* textureY : y position on the original image to start (defaults to 0)
-* textureWidth : width of each frame (defaults to null, that is, full width of the sprite sheet)
-* textureHeight : height of each frame (defaults to null, that is, full height of the sprite sheet)
-* removeOnFinish : a bool indicating if this AnimationComponent should be removed when the animation has reached its end
-
-So, in our example, we are saying that we have 8 frames for our player animation, and they are displayed in a row. So if the player height is also 16 pixels, the sprite sheet is 128x16, containing 8 16x16 frames.
-
 This constructor makes creating an Animation very easy using sprite sheets.
 
+In which you pass the image instance, and the frame data, which contains some parameters which can be used to describe the animation. Check the documentation on the constructors available on `SpriteAnimationFrameData` class to see all the parameters.
```

Comment:
"In which" can't be used since the subject is in the previous paragraph, I would write "In the constructor you pass an image instance, ..."

Third "which" should be "that"
-> "which contains some parameters that can ..."